container.go

```diff
@@ -860,20 +860,38 @@ func (c *Client) PutContainerArchive(id string, opts PutContainerArchiveOptions)
 	})
 }
 
-// CopyFromContainerOptions is the set of options that can be used when copying
-// files or folders from a container.
+// GetContainerArchiveOptions is the set of options that can be used when
+// downloading resources from a container.
 //
-// See https://goo.gl/4L7b07 for more details.
+// See https://goo.gl/KnZJDX for more details.
+type GetContainerArchiveOptions struct {
+	OutputStream io.Writer `json:"-" qs:"-"`
+	Path         string    `qs:"path"`
+}
+
+// REVIEW: 'GetContainerArchive' Needs a better name since there's also 'ExportContainer', confusing?!
+// 		 IMO change both PUT/GET funcs to:  'DownloadFromContainer' & 'UploadToContainer'
+
+// GetContainerArchive downloads a tar archive of files or folders in a container.
+//
+// See https://goo.gl/KnZJDX for more details.
```

Comment:
I like the idea of using DownloadFromContainer and UploadToContainer, feel free to change both methods (I know it will break the later, but I think it's fresh enough).